Source/WebCore/Modules/mediastream/gstreamer/GStreamerWebRTCUtils.cpp

```diff
@@ -776,6 +788,208 @@ void forEachTransceiver(const GRefPtr<GstElement>& webrtcBin, Function<bool(GRef
     }
 }
 
+#define CRLF "\r\n"_s
+
+class SDPStringBuilder : public StringBuilder {
```

Comment:
LGTM all but this. I would not create as inheriting from StringBuild. I would use StringBuilder inside SDPStringBuilder, but not deriving.